### DISCUSSION TITLE
Grouped by country,Taiwan is not a country,but a province of China.

### DISCUSSION THREAD

**Discussion by jsbyddy** [iptv-org/community#623] [Ideas]
### Describe your issue

Grouped by country,Taiwan is not a country,but a province of China.

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by jsbyddy** [comment]
And Hong Kong and Macao are not countries too.

**Comment by PopeyeTheSai10r** [comment]
Feel free to create a PR with changes.
